packages/generator-langium/src/index.ts

```diff
@@ -6,8 +6,8 @@
 
 import Generator from 'yeoman-generator';
 import _ from 'lodash';
+import chalk from 'chalk';
```

Comment:
I assume chalk is imported from `yeoman-generator`? We should add an explicit dependency on that in the `generator-langium` package.